.dockerignore

```diff
@@ -3,3 +3,4 @@
 **/.terraform/
 target/
 !target/libra-node-builder/libra-node
+**/*Dockerfile
```

Comment:
This breaks layer caching between the different images. The Dockerfile getting used is added to the context, which changes the context hash for the different image builds.

Reply:
so including all the docker files as source (for the copy) is a means to share layers between docker builds with different different docker files, but with identical lines?  Got it.

What other ways of achieving this exist?   Perhaps an image with all compiled code in it used as a FROM?   Looking for ways to share layers w/o having a change to a docker file force cargo to recompile.

Also contemplating building nightlies images that retain the cargo target dirs for reuse (much the same way local build works, or the shadow image's target is reused in this pr).

Reply:
IIUC, this should avoid rebuilding the expensive rust build step, esp when iterating changes on Docker files.